oauth2/oauth2-core/src/main/java/org/springframework/security/oauth2/core/endpoint/OAuth2AuthorizationRequest.java

```diff
@@ -401,5 +398,11 @@ private String buildAuthorizationRequestUri() {
 					.build()
 					.toUriString();
 		}
+
+		private LinkedHashSet<String> toLinkedHashSet(String... scope) {
+			final LinkedHashSet<String> result = new LinkedHashSet<>();
```

Comment:
To conform with other Spring Security code, please remove the `final` keyword from in front of local variables.

Reply:
Removed.